models.rdf

```diff
@@ -66,6 +66,14 @@
       <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
     </rdfs:Class>
 
+    <rdfs:Class rdf:about="http://pcdm.org/models#FileSet">
```

Comment:
Then needs definition at the abstract level as to what sorts of metadata it can have.  Can it have both technical and descriptive, or only one and if so which?